Author: Daiki Ueno <ueno@unixuser.org>
Description: Fix No package 'ibus-1.0' found error for IBus_1_0_gir
 .
 Osamu Aoki <osamu@debian.org> writes:
 > Package ibus-1.0 was not found in the pkg-config search path.
 > Perhaps you should add the directory containing `ibus-1.0.pc'
 > to the PKG_CONFIG_PATH environment variable
 > No package 'ibus-1.0' found
 > Any idea what is needed?  I have pushed my git repo to the alioth.
 .
 Sorry, this is presumably my bad.  I'll upstream the attached fix.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -170,7 +170,7 @@
     ibusenumtypes.h             \
     $(NULL)
 IBus-1.0.gir: $(libibus) Makefile
-IBus_1_0_gir_SCANNERFLAGS = --pkg=ibus-1.0 $(IBUS_GIR_SCANNERFLAGS)
+IBus_1_0_gir_SCANNERFLAGS = --pkg-export=ibus-1.0 $(IBUS_GIR_SCANNERFLAGS)
 IBus_1_0_gir_INCLUDES = GLib-2.0 GObject-2.0 Gio-2.0
 IBus_1_0_gir_LIBS = $(libibus)
 IBus_1_0_gir_FILES = $(addprefix $(srcdir)/,$(introspection_files))
